sway-lib-core/src/primitive_conversions.sw

```diff
@@ -18,7 +18,7 @@ impl u64 {
     /// ```
     pub fn as_u256(self) -> u256 {
         let input = (0u64, 0u64, 0u64, self);
-        asm(input: input) {
+        asm( input: input ) {
```

Comment:
The spacing around asm arguments looks odd. Was that intentional?

Reply:
Are we able to just skip asm blocks in the formatter for now as we discussed during the sync yesterday?

Reply:
asm blocks looks fine now, marking this as resolved.